src/dune_rules/pkg_rules.ml

```diff
@@ -432,30 +450,40 @@ module Action_expander = struct
       | Stublibs -> Path.relative roots.lib_root "stublibs"
       | Misc -> assert false
 
+    let sys_poll_var key =
+      let+ v = Memo.Lazy.force key in
+      match v with
+      | Some v -> [ Value.String v ]
+      | None ->
+        (* TODO *)
```

Comment:
Could you clarify this TODO a little? What should be done here ideally? When can we expect `None` here?

Reply:
It I understand the OPAM codebase correctly, it will eventually call `opt_eval_to_bool` which converts `None` to `false` https://github.com/ocaml/opam/blob/f67683cb49171615a475b0b7493b3bcc67a0b585/src/format/opamFilter.ml#L392-L395 for all unknown variables.

In our case we would get `None` e.g. if our platform is some kind of Unix but `uname` can't be found.

Reply:
Thanks. Could you include it in the comment though.